sci-electronics/kicad/files/kicad-boost-1.61.patch

```diff
@@ -0,0 +1,347 @@
+--- a/common/tool/tool_manager.cpp
```

Comment:
This patch looks kinda big. Doesn't repoman complain about it? Also, is it upstreamed? If so, please add a link to upstream commit/bug/patch.